README.md

```diff
@@ -170,7 +170,35 @@ questions:
 
 **subquestions**: `subquestions[]` cannot contain `subquestions` or `show_subquestions_if` keys, but all other keys in the above table are supported. 
 
-### 6. Test your changes
+### 6. Update package to track new upstream (Maintenance)
+
+There are two ways you can update a package, one is to track a new updated upstream chart
+and the other is to do small modifications/fixes.
+
+#### Update package to track a new upstream chart
+
+Update the `url` in `package.yaml` to reference the new upstream chart and `make prepare CHART={CHART_NAME}` 
```

Comment:
We should add a note about resetting the `packageVersion` as well when you rebase to a new upstream.